### PR TITLE
update graph_objs.py

### PR DESCRIPTION
**Pull request by cldougl**
update for https://github.com/plotly/streambed/pull/10150
cc @jackparmer @chriddyp @Kully

### CODE REVIEW

File: plotly/version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = '2.0.16'
+__version__ = '2.0.17'
```
Comment (chriddyp) [inline plotly/version.py]: I think we've actually been doing semver wrong. `Table` is a new feature, so I believe that this should be `2.1.0` (http://semver.org/)

**Comment by chriddyp** [comment]
was this generated by a script or done by hand?

File: plotly/version.py (hunk 1 above)
Comment (cldougl) [reply plotly/version.py]: oh I thought we were only bumping if it was a new plotly.py specific feature

**Comment by cldougl** [comment]
by script: `make update_default_schema`
https://github.com/plotly/plotly.py/blob/master/makefile#L19

**Comment by jackparmer** [comment]
💃  looks like it will work to me. Once it's on pip I'm happy to take it for a spin on `streambed`

**Pull request merged**